rsmtool/rsmeval.py

```diff
@@ -10,30 +10,34 @@
 :organization: ETS
 """
 
-
-import argparse
 import logging
 import os
 import sys
 
 from os import listdir
 from os.path import abspath, exists, join
 
-from . import VERSION_STRING
 from .analyzer import Analyzer
 from .configuration_parser import configure
 from .preprocessor import FeaturePreprocessor
 from .reader import DataReader
 from .reporter import Reporter
+from .utils.commandline import generate_configuration, setup_rsmcmd_parser
+from .utils.constants import VALID_PARSER_SUBCOMMANDS
 from .utils.logging import LogFormatter
 from .writer import DataWriter
 
 
-def run_evaluation(config_file_or_obj_or_dict, output_dir):
+def run_evaluation(config_file_or_obj_or_dict,
+                   output_dir,
+                   overwrite_output=False):
```

Comment:
Yikes. This default `False` setting will break a lot of my code since previously you didn't need to worry about overwriting when calling API. It probably makes sense to keep it the way you did it, but we'll want to make sure we document it: could you this to your PR description for when I am writing the release note?

Reply:
Updated the PR description.